### PR TITLE
hero section added

### PR DESCRIPTION
**Pull request by PakizeBozkurt**
created hero section

#28

### CODE REVIEW

File: langjo-frontend/src/Home.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-
+import Hero from "./components/Hero/Hero";
 
 function Home() {
   return (
     <div>
       <LevelTest />
       <BeginnerLevel />
       <BlocksCards />
+      <Hero />
```
Comment (PakizeBozkurt) [inline langjo-frontend/src/Home.js]: <Hero /> should be in first one, put it on line 10 please.

**Review by PakizeBozkurt — COMMENTED** [review]
You did great effort! Thank you Kavita, see the my comments on Home.js section.

File: langjo-frontend/src/Home.js (hunk 1 above)
Comment (kavitappatil) [reply langjo-frontend/src/Home.js:13]: Done

Comment (kavitappatil) [reply langjo-frontend/src/Home.js]: Done, thanks

**Pull request merged**